src/giftCards/GiftCardsList/providers/GiftCardListDialogsProvider/GiftCardListDialogsProvider.tsx

```diff
@@ -78,11 +79,12 @@ const GiftCardListDialogsProvider: React.FC<GiftCardListDialogsProviderProps> =
   return (
     <GiftCardListDialogsContext.Provider value={providerValues}>
       {children}
-      <GiftCardCreateDialog
-        open={isDialogOpen(CREATE)}
-        onClose={onClose}
-        refetchQueries={[GIFT_CARD_LIST_QUERY]}
-      />
+      <Dialog open={isDialogOpen(CREATE)} maxWidth="sm">
+        <GiftCardCreateDialogContent
+          onClose={onClose}
+          refetchQueries={[GIFT_CARD_LIST_QUERY]}
+        />
+      </Dialog>
       <GiftCardListPageDeleteDialog
```

Comment:
What about the rest of them? Will it be refactored in later PR?

Reply:
After further investigation I don't think it's a viable idea, for example `GiftCardListPageDeleteDialog` uses a bit differently handled `ActionDialog` instead - it would be required to pass certain things up (`onConfirm, disabled, ...`) to the Provider. `GiftCardBulkCreateDialog` has additional dialog underneath that is handled by the component. I'll push an update with the `GiftCardExportDialog` component

Reply:
It's fine as long as no internal state is used there. Though, I'd still refactor them to use the same pattern underneath if you want them as a single component

Reply:
The dialogs I haven't changed to new pattern use a lot of internal state because of they have complex logic underneath to eg. display 3 different dialogs one after the other. To "fix" it we would have to still bring the state up to the provider (so it wouldn't be affected by the mui dialog behaviour anyway).